README.md

```diff
@@ -186,7 +191,7 @@ Host h
 Note that the cpu command is itself a 9p server; i.e., your instance of cpu runs your server. The remote
 cpu server may run as root, but all file accesses happen locally as you. Hence,
 the cpu command does not grant greater access to the local machine than you already possess.
-I.e., there is no privilege escalation.
+i.e., there is no privilege escalation.
```

Comment:
you sure about this? i.e. is starting a new sentence.

Reply:
I don't even remember making that change, I wonder if vscode was being clever....

Reply:
fixed